### PR TITLE
upload 20220421 HW

### PR DESCRIPTION
**Pull request by sunho1**

### CODE REVIEW

File: Sunho/20220421/BOJ_11399_ATM.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+			
+			int num=pq.poll();
+			sum+=num*cnt;
+			cnt--;
+			
+		}
+		
+		System.out.println(sum);
+	}
+}
```
Comment (hamakim94) [inline Sunho/20220421/BOJ_11399_ATM.java:36]: pq 굳드!!

**Review by konrini — COMMENTED** [review]
👍

File: Sunho/20220421/BOJ_11659_구간합4.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+			
+			sb.append(sum).append("\n");
+			
+		}
+		
+		System.out.println(sb.toString());
+		
+		
+	}
+}
```
Comment (hamakim94) [inline Sunho/20220421/BOJ_11659_구간합4.java:36]: 굳!!!!

**Review by hamakim94 — COMMENTED** [review]
굳드!!

**Comment by soomi1015** [comment]
잘 봤습니당~👍🏻

**Pull request merged**